solara/website/components/sidebar.py

```diff
@@ -34,58 +37,61 @@ def Sidebar():
                                 ),
                             }
                         ],
-                        value=router.path.startswith("/documentation/" + route.path),
+                        value=top_level_expanded,
+                        eager=True,  # better for SEO
                     ):
                         for item in route.children:
-                            if item.path == "/":
-                                continue
+                            label = item.label
+                            if item.path == "/" and route.path in ["examples", "api", "components"]:
+                                # the 'homepage' of the subpage are named Overview
+                                label = "Overview"
+                            path_sub = "/documentation/" + route.path + "/" + item.path
+                            sub_should_be_expanded = router.path.startswith(path_sub)
                             if item.children != [] and any([c.label is not None and c.path != "/" for c in item.children]):
                                 with solara.v.ListGroup(
                                     v_slots=[
                                         {
                                             "name": "activator",
                                             "children": solara.v.ListItemTitle(
-                                                children=[item.label],
+                                                children=[label],
                                             ),
                                         }
                                     ],
                                     sub_group=True,
                                     no_action=True,
-                                    value=router.path.startswith("/documentation/" + route.path + "/" + item.path),
+                                    eager=True,  # better for SEO
+                                    value=sub_should_be_expanded,
                                 ):
                                     for subitem in item.children:
-                                        # skip pages that are only used to demonstrate Link or Router usage
-                                        if subitem.path == "/" or subitem.label is None:
+                                        # skip the 'homepage' of the examples only
+                                        if subitem.path == "/" and route.path == "examples":
```

Comment:
```suggestion
                                        if subitem.path == "/" and route.path != "getting_started":
```

This causes for a link to an invalid route ("Not supposed to be rendered") to be present under each subsection of the components and api sections, so those should at least be excluded.

Also, a lot of the subsection overview pages under advanced are also very half baked (see [here](https://solara.dev/documentation/advanced/development/) for an example), so I think we should either improve those, or exclude them as well (as I've done in the suggestion).

Reply:
Seeing as we want to add content to the "enterprise" overview page, I suppose we can improve the content of the other advanced overview pages in that PR and merge at the same time?